liberty-managed/README.md

```diff
@@ -1,12 +1,14 @@
-# Arquillian Liberty Managed Documentation
+# Arquillian Liberty Managed
```

Comment:
Since the EE 8 documentation is named README.md it will be rendered by deafult if people navigate to the directories. ex https://github.com/OpenLiberty/liberty-arquillian/tree/master/liberty-managed

A question i have is if we want the EE 8 or EE 9 docs rendered by default for people navigating to the /liberty-managed or /liberty-remote directoried.

Reply:
I think it makes the most sense for the EE 8 docs to appear by default since I imagine most users will still be using Java EE 8 for quite a while

Reply:
Yeah that sounds good to me. Just wanted to bring up the question.

Reply:
Alternatively, we can move both the EE 8 and EE 9 documentation into a "docs" folder under liberty-managed/liberty-remote respectively.